remove outdated info on tokens (#1793)
conda-forge/conda-forge.github.io: src/maintainer/knowledge_base.rst

So make sure that you **tag** your package in such a way that the package name
that conda-build spits out will sort the package uploaded with an ``rc`` label
higher than the package uploaded with the ``dev`` label.

How to update your feedstock token?
====================================
To reset your feedstock token and fix issues with uploads, follow these steps:

> 1. Create a new text file in ``token_reset`` directory. 
> 2. Add the name of your feedstock in the text file. While adding the name make sure to use name without adding "-feedstock". For example: for ``python-feedstock``, you write just ``python``.

Reviewer: ```suggestion
1. Create a new text file in the ``token_reset`` directory. 
2. Add the name of your feedstock in the text file. While adding the name, don't add "-feedstock" to the end of it. For example: for ``python-feedstock``, just add ``python``.
```

that conda-build spits out will sort the package uploaded with an ``rc`` label
higher than the package uploaded with the ``dev`` label.

How to update your feedstock token?
====================================
To reset your feedstock token and fix issues with uploads, follow these steps:

> 1. Create a new text file in the ``token_reset`` directory.

Reviewer 2: ```suggestion
1. Create a new text file in the ``token_reset`` directory of the ``conda-forge/admin-requests`` repo.
```
